backend/web/router/data_router.py

```diff
@@ -184,14 +184,10 @@ def region_temp():
     and noaa.tid = rft.tid
     GROUP BY date(rft.reftime)
     '''
+    return make_response(jsonify(
+        fill_series(date_series,
+                    Connection.sql_execute(query.format(region_id=region_id, timestamp=timestamp_str, days=days)))))
```

Comment:
change it to f-string instead of str.format

Reply:
I have changed them according to your comments.